pkgs/applications/graphics/gcolor3/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, gnome3, libtool, intltool, pkgconfig, gtk3, hicolor_icon_theme, wrapGAppsHook } :
+
+let
+  version = "2.2";
+in stdenv.mkDerivation {
+  name = "gcolor3-${version}";
+
+  src = fetchFromGitHub {
+    owner = "hjdskes";
+    repo = "gcolor3";
+    rev = "v${version}";
+    sha256 = "1rbahsi33pfggpj5cigy6wy5333g3rpm8v2q0b35c6m7pwhmf2gr";
+  };
+
+  nativeBuildInputs = [ gnome3.gnome_common libtool intltool pkgconfig hicolor_icon_theme wrapGAppsHook ];
+
+  buildInputs = [ gtk3 ];
+
+  configureScript = "./autogen.sh";
+
+  meta = {
+    description = "A simple color chooser written in GTK3";
+    homepage = https://hjdskes.github.io/projects/gcolor3/;
+    license = stdenv.lib.licenses.gpl2;
+    maintainers = with stdenv.lib.maintainers; [ jtojnar ];
+    platforms = stdenv.lib.platforms.unix;
```

Comment:
It does not build on macOS: https://travis-ci.org/NixOS/nixpkgs/jobs/282964506
You can limit the platforms to Linux.

Reply:
Fixed.